### PR TITLE
Add Pulsar Community Weekly Update - 2020-07-04 ~ 2020-07-10

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on.
+
+### Notable Feature
+
+- [Broker] Use CGroup CPU usage when present.
+
+    https://github.com/apache/pulsar/pull/7475
+    
+- [C++ Client] Add support for oauth2 auth.
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-10-pulsar-weekly.md:19]:
```suggestion
- [C++ Client] Add support for OAuth2 authentication.
```

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    https://github.com/apache/pulsar/pull/7467
+    
+- [Function] Differentiate authorization between source/sink/function operations.
+
+    https://github.com/apache/pulsar/pull/7466
+    
+- [Function] Allow function rebalance to be run periodically.
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-10-pulsar-weekly.md:27]:
```suggestion
- [Function] Allow function rebalance to run periodically.
```

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    https://github.com/apache/pulsar/pull/7449
+    
+### Notable Bug Fix
+
+- [Gep Replication] Handle `NoSuchElemetnException` when update geo replication policy.
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-10-pulsar-weekly.md:33]:
```suggestion
- [Gep Replication] Handle `NoSuchElemetnException` when updating geo replication policy.
```

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    https://github.com/apache/pulsar/pull/7459
+    
+- [Java Client] Fix batch ackset recycled multiple times.
+
+    https://github.com/apache/pulsar/pull/7409
+
+### Event / News
+
+- [Pulsar Summit] The first-ever Pulsar Summit Virtual Conference 2020 was held on June 17 - 18. This two-day digital event featured more than 30 sessions from top Pulsar contributors and developers.
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-10-pulsar-weekly.md:63]: shall we add this again?

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md (hunk 3 above)
Comment (Jennifer88huang) [reply content/2020/2020-07/2020-07-10-pulsar-weekly.md:33]: shall we refine the title or keep it consistent with the PR title?

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md (hunk 4 above)
Comment (Anonymitaet) [reply content/2020/2020-07/2020-07-10-pulsar-weekly.md:63]: Yes, like we promoting Summit on Twitter this week.

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md (hunk 3 above)
Comment (Anonymitaet) [reply content/2020/2020-07/2020-07-10-pulsar-weekly.md:33]: Think this description is clear enough.

File: content/2020/2020-07/2020-07-10-pulsar-weekly.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+id: "2020-07-10-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-07-04 ~ 2020-07-10
+
+This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on.
+
+### Notable Feature
+
+- [Broker] Use CGroup CPU usage when present.
```
Comment (Jennifer88huang) [inline content/2020/2020-07/2020-07-10-pulsar-weekly.md:15]: could you please update the title as well?

**Pull request merged**